src/Symfony/Component/Console/Command/Command.php

```diff
@@ -220,7 +220,10 @@ public function run(InputInterface $input, OutputInterface $output)
             if (function_exists('cli_set_process_title')) {
                 if (!@cli_set_process_title($this->processTitle)) {
                     if ('Darwin' === PHP_OS) {
-                        $output->writeln('<comment>Running "cli_get_process_title" as an unprivileged user is not supported on MacOS.</comment>');
+                        $output->writeln(
```

Comment:
Usually we keep everything on a single line.

Reply:
k. wasn't sure about line lengths.